README.md

```diff
@@ -3,10 +3,26 @@ This repository hosts the Solo.io API definitions. It is intended as a read-only
 APIs are the projects that own them. The goal of this repository is to allow for projects to consume an API 
 without having to depend on the project that owns it.
 
-## Gloo Apis
-Syncing the Gloo APIs is currently a semi-automated process and is done with 2 github actions, requiring manual approval in order to merge into master.
+## Gloo Edge APIs
+For each release of a Gloo Edge, we want to freeze the solo-apis dependency by creating a unique tag tied to the dependent release. For example, if Gloo Edge `v1.8.4` is released, we want to create a `gloo-v1.8.4` tag in solo-apis with a read-only mirror of the API. 
 
-1. In the gloo repo, a github action will run on each Gloo OS release. It will run `./hack/sync-gloo-apis.sh; make generate -B` in solo-apis and push to the `update-gloo-apis` branch in solo-apis.
-2. In the solo-apis repo, each push to `update-gloo-apis` will automatically create a PR.
-3. A member of the solo-io org must approve the pull request.
-4. Once the PR has merged into master, a tag should be created off of master corresponding to the Gloo tag which was used.
+In order to keep API definitions in sync, we automate the process of syncing long term support (LTS) branches. To keep the LTS branches unique, we prefix them with the `gloo`. For example, in gloo-edge the `1.8.x` LTS branch is tracked by the `gloo-1.8.x` LTS branch in solo-apis.
```

Comment:
```suggestion
In order to keep API definitions in sync, we automate the process of syncing long term support (LTS) branches. To keep the LTS branches unique, we prefix them with the `gloo`. For example, in gloo-edge the `v1.8.x` LTS branch is tracked by the `gloo-v1.8.x` LTS branch in solo-apis.
```

Reply:
In addition I think
> we prefix them with the `gloo`

should become something like
> we prefix them with the dependent repo name (ie `gloo`).